crates/holochain/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Refactor conductor methods `enable_app`, `disable_app`, `uninstall_app` and `initialize_conductor` to directly manage cells instead of using state machine code.
 - **BREAKING CHANGE**: `AdminRequest::EnableApp` fails when creating the app's cells fails and returns the first error that occurred. In case of success the enabled app info is returned.
 - Remove state machine functions from conductor, which have been replaced by functions that process the necessary steps directly.
+- **BREAKING CHANGE**: Remove unused function `get_dependency_apps`.
```

Comment:
I think you got a typo here :stuck_out_tongue: . Just kidding.

Reply:
Yeah - I didn't want to repeat the misspelled name here =D